### PR TITLE
test(e2e,playwright): <lower-jaw>

### PR DESCRIPTION
**Pull request by AidanLDev**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705

### CODE REVIEW

**Comment by AidanLDev** [comment]
camperbot cant code

**Comment by AidanLDev** [comment]
I struggled to write tests for this component because I don't fully understand it, will need to be finished off by someone who knows the code base as the coverage isn't very high

**Comment by gikf** [comment]
Hey @AidanLDev

We would love to be able to merge your changes, but it looks like you have some merge conflicts. ⚠️

Once you resolve these conflicts, we will be able to review your PR and merge it. 😊

---

If you're not familiar with the merge conflict process, feel free to look over GitHub's guide on ["Resolving a merge conflict"](https://help.github.com/articles/resolving-a-merge-conflict-on-github/). 🔍️

Also, it's good practice on GitHub to write a brief description of your changes when creating a PR. 📝

**Comment by AidanLDev** [comment]
Thanks for the heads up, I'll resolve and re-test then let you know :) @gikf

**Comment by AidanLDev** [comment]
Checks are still ongoing but conflict should now be sorted 😄

File: e2e/lower-jaw.spec.ts [hunk 1]
```diff
@@ -37,3 +37,27 @@ test('Click on the "Reset" button', async ({ page }) => {
...
 });
+
+test('Should render UI correctly', async ({ page }) => {
+  const codeCheckButton = page.getByRole('button', {
+    name: 'Check Your Code'
+  });
+  const lowerJawTips = page.getByTestId('failing-test-feedback');
+  await expect(codeCheckButton).toHaveText('Check Your Code (Ctrl + Enter)');
+  await expect(lowerJawTips).toHaveCount(0);
+  await codeCheckButton.click();
```
[the remark this answers is not in the document]
Comment (AidanLDev) [reply e2e/lower-jaw.spec.ts:48]: Good point, I will remove the click action

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 Thank you for your contribution @AidanLDev!

**Review by moT01 — APPROVED** [review]
LGTM @AidanLDev 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**